apix-integrationtests/build.gradle

```diff
@@ -67,7 +67,9 @@ subprojects {
     alfrescoDynamicExtensions {
         repository {
             endpoint {
-                protocol = "http"
+                protocol = project.hasProperty('protocol') ? project.protocol : 'http'
```

Comment:
It will always be http, which is also the default, so you can leave this configuration off.